website/content/en/docs/faqs/_index.md

```diff
@@ -234,7 +234,25 @@ Though this is a bug with controller-gen which is used by Operator SDK to genera
 ## What is the bundle limit size? Was this amount increased?
 
 Bundles have a size limitation because their manifests are used to create a configMap, and the Kubernetes API does not 
-allow configMaps larger than `~1MB`. However, from [OLM](https://github.com/operator-framework/operator-lifecycle-manager) version `v0.19.0` and [OPM](https://github.com/operator-framework/operator-registry) `1.17.5`, these values were increased to `~4MB` because we are compressing them. ([More info](https://github.com/operator-framework/operator-registry/pull/685)).
+allow configMaps larger than `~1MB`. Beginning with [OLM](https://github.com/operator-framework/operator-lifecycle-manager) version `v0.19.0` 
+and [OPM](https://github.com/operator-framework/operator-registry) `1.17.5`, 
+these values are now compressed accommodating larger bundles. ([More info](https://github.com/operator-framework/operator-registry/pull/685)).
 
 The change to allow bigger bundles from [OLM](https://github.com/operator-framework/operator-lifecycle-manager) version `v0.19.0` only impacts the full bundle size amount. 
 Any single manifest within the bundle such as the CRD will still make the bundle uninstallable if it exceeds the default file size limit on clusters (`~1MB`).
+
+## The size of my Operator bundle is too big. What can I do?
+
+If your bundle is too large, there are a few things you could try.
+
+* Consider reducing the number of [CRD versions][k8s-crd-versions] supported in your Operator?
+* Deprecate or reduce the number of supported API versions? It is a good idea to have a clear
```

Comment:
what's the difference between "Consider reducing the number of CRD versions" and "Deprecate or reduce the number of supported API versions" ?

Reply:
@bparees that was my edit. The original text was as follows:

Reply:
I see, so by "CRD versions" you meant "distinct CRDs" and not "versions of a given CRD/type"

In that case, "Consider reducing the number of CRD types" might be clearer.  Though as i said on the earlier edits, i'm pretty reluctant to tell anyone to adapt their preferred API to address OLM size constraints.  That is definitely the tail wagging the dog.

Reply:
@bparees I think the first one suggests reduce the the operator's API versions and the second bullet suggests having a deprecation plan that will help you do the first one. It was original a single paragraph but I broke it up into bullets.

Reply:
the current context+formatting imply to me that the two bullets are distinct techniques/suggestions for reducing size, not that the second one is details on how to accomplish the first one.

how about (yes it goes back to one big paragraph, but i've tried to clean it up a bit):

```
If your bundle is too large, there are a few things you can try:

  * Reducing the number of [CRD versions](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#specify-multiple-versions) supported in your Operator by deprecating and then removing older API versions. It is a good idea to have a clear plan for deprecation and removal of old CRDs versions when a new ones get added, see [Kubernetes API change practices](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md). Also, refer to the [Kubernetes API conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md).
  * Reduce the verbosity of your API documentation. (We do not recommend eliminating documenting the APIs)
```

Reply:
Done as. suggested @bparees @jmrodri